Packs/MailListener/ReleaseNotes/1_0_54.md

```diff
@@ -0,0 +1,6 @@
+
+#### Integrations
+
+##### Mail Listener v2
+
+- Fixed an issue where an error occurred when an email attachment was not parsed correctly.
```

Comment:
```suggestion
- Fixed an issue where an email attachment was not parsed correctly.
```
An error will still be raised if the attachment could not be parsed. The fix here in case of email that has an email attachment that wasn't parse correctly.